portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/TestDB.java

```diff
@@ -12,9 +12,10 @@
  * details.
  */
 
-package com.liferay.portal.dao.db;
+package com.liferay.portal.dao.sql.transformer;
```

Comment:
@dantewang @ChiZelin 
I remembered that we have an issue about ee test db classes, if we move the base classes to portal-impl unit test package, modules can not see it. So we are clear now?

Tina.